[2022-09-26T12:53:50.646+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.view_data scheduled__2022-06-01T00:00:00+00:00 [queued]>
[2022-09-26T12:53:50.678+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.view_data scheduled__2022-06-01T00:00:00+00:00 [queued]>
[2022-09-26T12:53:50.678+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T12:53:50.679+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-09-26T12:53:50.679+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T12:53:50.730+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): model.migrate_to_dbt.view_data> on 2022-06-01 00:00:00+00:00
[2022-09-26T12:53:50.743+0000] {standard_task_runner.py:54} INFO - Started process 4066 to run task
[2022-09-26T12:53:50.749+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'model.migrate_to_dbt.view_data', 'scheduled__2022-06-01T00:00:00+00:00', '--job-id', '2127', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmppwwotydo']
[2022-09-26T12:53:50.750+0000] {standard_task_runner.py:83} INFO - Job 2127: Subtask model.migrate_to_dbt.view_data
[2022-09-26T12:53:50.752+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dbt_dag.py
[2022-09-26T12:53:50.915+0000] {task_command.py:384} INFO - Running <TaskInstance: dbt_dag.model.migrate_to_dbt.view_data scheduled__2022-06-01T00:00:00+00:00 [running]> on host 3c654011b3f0
[2022-09-26T12:53:51.066+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=model.migrate_to_dbt.view_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-01T00:00:00+00:00
[2022-09-26T12:53:51.067+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-09-26T12:53:51.068+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            cd /opt/***/migrate_to_dbt &&\n            dbt --no-write-json run --target prod --models view_data\n            ']
[2022-09-26T12:53:51.088+0000] {subprocess.py:86} INFO - Output:
[2022-09-26T12:53:51.091+0000] {subprocess.py:93} INFO - /bin/bash: line 3: dbt: command not found
[2022-09-26T12:53:51.092+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-09-26T12:53:51.117+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-09-26T12:53:51.124+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=dbt_dag, task_id=model.migrate_to_dbt.view_data, execution_date=20220601T000000, start_date=20220926T125350, end_date=20220926T125351
[2022-09-26T12:53:51.152+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 2127 for task model.migrate_to_dbt.view_data (Bash command failed. The command returned a non-zero exit code 127.; 4066)
[2022-09-26T12:53:51.203+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-26T12:53:51.315+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
